sensor/common/compliance/service_impl.go

```diff
@@ -47,6 +47,10 @@ type serviceImpl struct {
 	stopper     set.Set[concurrency.Stopper]
 }
 
+func (s *serviceImpl) Name() string {
+	return fmt.Sprintf("%T", s)
```

Comment:
I think brain-fueled names would provide more value, specially where the types are generic words, because places where they are used such as logs would become more readable:

`serviceImpl is online`
vs
`compliance service is online`

Reply:
Yes, this is better but requires more work. See [my reply for Tomek](https://github.com/stackrox/stackrox/pull/16028#discussion_r2207164502)